main.sh

```diff
@@ -73,7 +73,7 @@ cluster_up() {
 	KUBECONFIG=$kubeconfig kubectl config view --merge --flatten >all-in-one-kubeconfig.yaml
 	mv -f all-in-one-kubeconfig.yaml "${KUBECONFIG_ROOT_DIR}/config"
 
-	export KUBECONFIG="${KUBECONFIG_ROOT_DIR}/config"
+	export KUBECONFIG="${KUBECONFIG_ROOT_DIR}/$KEPLER_KUBECONFIG"
```

Comment:
The previous value exported here was causing issues during grafana deployment when invoking the kubectl command. Setting it to the variable `${KUBECONFIG_ROOT_DIR}/$KEPLER_KUBECONFIG` fixes this issue